service/src/main/java/uk/gov/hmcts/reform/fpl/service/CaseManagementOrderService.java

```diff
@@ -26,6 +50,20 @@ public CaseManagementOrder addAction(CaseManagementOrder order, OrderAction orde
             .build();
     }
 
+    public void progressCMOToAction(CaseDetails caseDetails, CaseManagementOrder order) {
```

Comment:
It this method used anywhere?

Also, `caseDetails.getData()` can be extracted to variable before switch.

Reply:
Method is not used so removed

Reply:
It has been replaced with the functionality brought in by the CaseManagementOrderProgression service and controller.